technic/init.lua

```diff
@@ -4,6 +4,14 @@ technic = rawget(_G, "technic") or {}
 technic.plus = true
 technic.version = 1.2
 
+technic.sounds = {}
+if minetest.get_modpath("default") then
+   technic.sounds = default
+end
+if minetest.get_modpath("mcl_sounds") then
+   technic.sounds = mcl_sounds
+end
```

Comment:
I think it would still be better to keep using default as a fallback (even if it is not coming from a mod named default):
```suggestion
if minetest.get_modpath("mcl_sounds") then
   technic.sounds = mcl_sounds
else
   technic.sounds = default or {}
end
```
(fixed issues with previous suggestion)

Reply:
I don't understand the reason for doing this

Default is a mod part of a game (MTG) that continues to be used for historical reasons, and that will end up disappearing and being replaced by better games.

The only reason for this change would be to legitimize Default as an important mod that is above other games.

Reply:
> The only reason for this change would be to legitimize Default as an important mod that is above other games.

Reason is to ensure backwards compatibility by having fallback option like it was before.

Reply:
ok done